model/intent/record_eeg.py

```diff
@@ -1,32 +1,31 @@
-    parser.add_argument('--ip-protocol', type=int, help='ip protocol, check IpProtocolType enum', required=False,
-                        default=0)
-    parser.add_argument('--ip-address', type=str, help='ip address', required=False, default='')
-    parser.add_argument('--serial-port', type=str, help='serial port', required=False, default='')
-    parser.add_argument('--mac-address', type=str, help='mac address', required=False, default='')
-    parser.add_argument('--other-info', type=str, help='other info', required=False, default='')
-    parser.add_argument('--serial-number', type=str, help='serial number', required=False, default='')
-    parser.add_argument('--file', type=str, help='file', required=False, default='')
-    parser.add_argument('--actions', type=int, help='number of actions to record',
-                        required=True)
-    parser.add_argument('--sessions', type=int, help='number of sessions per action to record',
-                    required=False, default=2)
-    # board id by name or id
-    parser.add_argument('--board-id', type=str, help='board id or name, check docs to get a list of supported boards',
-                        required=True)
+    parser.add_argument('--timeout', type=int, required=False, default=0, help='timeout for device discovery or connection')
+    parser.add_argument('--ip-port', type=int, required=False, default=0, help='ip port')
+    parser.add_argument('--ip-protocol', type=int, required=False, default=0, help='ip protocol, check IpProtocolType enum')
+    parser.add_argument('--ip-address', type=str, required=False, default='', help='ip address')
+    parser.add_argument('--serial-port', type=str, required=False, default='', help='serial port')
+    parser.add_argument('--mac-address', type=str, required=False, default='', help='mac address')
+    parser.add_argument('--other-info', type=str, required=False, default='', help='other info')
+    parser.add_argument('--serial-number', type=str, required=False, default='', help='serial number')
+    parser.add_argument('--file', type=str, required=False, default='', help='file',)
+    parser.add_argument('--actions', type=int, required=True, help='number of actions to record')
+    parser.add_argument('--sessions', type=int, required=False, default=2, help='number of sessions per action to record')
+    parser.add_argument('--window-length', type=int, required=False, default=10, help='length in seconds of eeg data pulled per session')
+    parser.add_argument('--window-buffer', type=int, required=False, default=2, help='time in seconds before eeg data is recorded each session (delay after hitting enter)')
+    parser.add_argument('--overwrite', type=int, required=False, default=1, help='1 to overwrite/remove old recordings, 0 to add results as an additional data file')
+    parser.add_argument('--board-id', type=str, required=True, help='board id or name, check docs to get a list of supported boards')
```

Comment:
This area is a bit noisy too since it looks like all of the lines had their help arguments shifted to the end. I'll let this one slide though since I agree with moving the help argument to the end of the list, but I'd like it if you could provide a list of what was actually added here as its not easy to tell at a glance

Reply:
Right, my bad. I went and checked them all; no preexisting arguments were changed.
`window-length`, `window-buffer`, and `overwrite` were all added